fish_prompt.fish

```diff
@@ -911,6 +912,18 @@ function __bobthefish_prompt_hg -S -a hg_root_dir -a real_pwd -d 'Display the ac
     end
 end
 
+function __bobthefish_prompt_screen -S -d 'Display the screen name'
+    [ "$theme_display_screen" != 'yes' ]
+    and return
+
+    [ -z "$STY" ]
+    and return
+
+    __bobthefish_start_segment $color_screen
+    echo -ns (command echo $STY | cut -d. -f2) ' '
```

Comment:
we can probably do this entirely with the `string` builtin, saving ourselves a couple of calls out to core utils. what does $STY typically contain?

Reply:
done